src/main/java/com/imgix/URLHelper.java

```diff
@@ -184,6 +177,22 @@ public static String encodeURIComponent(String s) {
     return result;
   }
 
+  public static String encodeURI(String s) {
+    String result =
+        s.replaceAll("\\+", "%2B")
+            .replaceAll("\\:", "%3A")
+            .replaceAll("\\?", "%3F")
+            .replaceAll("\\#", "%23")
+            .replaceAll(" ", "%20")
+            .replaceAll("\\%21", "!")
+            .replaceAll("\\%27", "'")
+            .replaceAll("\\%28", "(")
+            .replaceAll("\\%29", ")")
+            .replaceAll("\\%7E", "~");
+
+    return result.replace("\\", "");
+  }
+
   public static String decodeURIComponent(String s) {
```

Comment:
I don't believe this is being used anymore.

Reply:
Sorry do you mean 
`public static String decodeURIComponent(String s) {` 
or
`    return result.replace("\\", "");`.

Reply:
No worries, I meant the `decodeURIComponent` method

Reply:
Ah good catch. Just so happens neither was needed 😆